modules/tinymce/src/themes/silver/test/ts/browser/editor/header/PromotionTest.ts

```diff
@@ -0,0 +1,64 @@
+import { ApproxStructure, Assertions, UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SelectorFind } from '@ephox/sugar';
+import { TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.header.PromotionTest', () => {
+  const promotionClass = 'tox-promotion';
+
+  context('promotion turned off', () => {
+    const hook = TinyHooks.bddSetup<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
```

Comment:
Maybe it's better to be specific here and pass in `false` even if you don't need to. Since it would be hard for someone just reading the test and the code in the theme to know that the default is changed inside the `bddSetup` call to `false`.